src/components/BusinessCard/BusinessCard.scss

```diff
@@ -0,0 +1,7 @@
+@use "../../styles/variables" as *;
+
+.button--Details{
+    color: $white;
+    background-color: $green-color-palette;
+    border-radius: 2rem;
+}
```

Comment:
this should be added to button.scss
and classname should follow BEM convention: `.button--details`